--- glame-1.0.0/configure.ac.orig	2003-04-27 10:55:43.000000000 +0200
+++ glame-1.0.0/configure.ac	2003-04-27 11:09:53.000000000 +0200
@@ -193,11 +193,12 @@
 
 if test x$glade = xyes; then
 dnl HACK
-	AC_MSG_WARN([Skipping libxml check due to possible conflicts with libglade])
-	XML_CFLAGS="$GNOME_INCLUDEDIR"
-	XML_LIBS="$GNOME_LIBS"
-	AC_SUBST(XML_CFLAGS)
-	AC_SUBST(XML_LIBS)
+	AC_MSG_WARN([Using libxml 1.x because of possible libxml2 conflicts with libglade])
+	AMG_PATH_XML(1.8.0, , [
+	AC_MSG_ERROR([You need at least libxml 1.8.0 to run glame!
+Hint: You also need to have the appropriate xml development package
+installed, usually called libxml-dev, libxmld, or similar.])
+])
 else
 AMG_PATH_XML2(2.0.0, , [
 	AMG_PATH_XML(1.8.0, , [
