diff -Nur glame-1.0.0/Makefile.am glame-1.0.0-patched/Makefile.am
--- glame-1.0.0/Makefile.am	Fri Nov  8 17:39:04 2002
+++ glame-1.0.0-patched/Makefile.am	Wed Apr 23 00:15:12 2003
@@ -1,4 +1,4 @@
-SUBDIRS = intl libltdl macros src doc data po
+SUBDIRS = intl macros src doc data po
 
 ACLOCAL_AMFLAGS = -I macros -I .
 
--- glame-2.0.1/configure.ac.orig	2005-03-28 19:04:18.000000000 +0200
+++ glame-2.0.1/configure.ac	2005-03-30 14:30:43.215748256 +0200
@@ -134,7 +134,7 @@
 
 
 AC_PROG_INSTALL
-AC_LIBLTDL_CONVENIENCE
+AC_LIBLTDL_INSTALLABLE
 dnl libtool pre-1.4 used INCLTDL, but LTDLINCL is the proper namespace.
 if test -z "${LTDLINCL}" -a ! -z "${INCLTDL}"; then
 	LTDLINCL="${INCLTDL}"
@@ -604,6 +604,5 @@
 
 AC_SUBST(LIBLTDL)
 AC_SUBST(LTDLINCL)
-AC_CONFIG_SUBDIRS(libltdl)
 
 AC_OUTPUT(Makefile macros/Makefile doc/Makefile doc/pix/Makefile data/Makefile data/pixmaps/Makefile src/Makefile src/filter/Makefile src/swapfile/Makefile src/hash/Makefile src/plugins/Makefile src/include/Makefile src/gui/Makefile src/glmid/Makefile intl/Makefile po/Makefile.in)
