diff -Nur glame-1.0.0/Makefile.am glame-1.0.0-patched/Makefile.am
--- glame-1.0.0/Makefile.am	Fri Nov  8 17:39:04 2002
+++ glame-1.0.0-patched/Makefile.am	Wed Apr 23 00:15:12 2003
@@ -1,4 +1,4 @@
-SUBDIRS = intl libltdl macros src doc data po
+SUBDIRS = intl macros src doc data po
 
 ACLOCAL_AMFLAGS = -I macros -I .
 
diff -Nur glame-1.0.0/configure.ac glame-1.0.0-patched/configure.ac
--- glame-1.0.0/configure.ac	Sat Apr 19 20:26:05 2003
+++ glame-1.0.0-patched/configure.ac	Wed Apr 23 00:16:22 2003
@@ -408,6 +408,5 @@
 
 AC_SUBST(LIBLTDL)
 AC_SUBST(LTDLINCL)
-AC_CONFIG_SUBDIRS(libltdl)
 
 AC_OUTPUT(Makefile macros/Makefile doc/Makefile doc/pix/Makefile data/Makefile data/pixmaps/Makefile src/Makefile src/filter/Makefile src/swapfile/Makefile src/hash/Makefile src/plugins/Makefile src/include/Makefile src/gui/Makefile src/glmid/Makefile  intl/Makefile po/Makefile.in )
diff -Nur glame-1.0.2/src/Makefile.am glame-1.0.2-patched/src/Makefile.am
--- glame-1.0.2/src/Makefile.am	Sun Dec 30 21:59:10 2001
+++ glame-1.0.2-patched/src/Makefile.am	Wed Apr 23 00:17:04 2003
@@ -15,6 +15,6 @@
 	filter/libfilter.la \
 	hash/libhash.la \
 	swapfile/libswapfile.la \
-	$(GLAME_LIBS) $(GUILE_LIBS) $(LIBLTDL) @LTLIBINTL@
+	$(GLAME_LIBS) $(GUILE_LIBS) -lltdl @LTLIBINTL@
 cglame_LDFLAGS = -export-dynamic -dlopen self
 
diff -Nur glame-1.0.2/src/gui/Makefile.am glame-1.0.2-patched/src/gui/Makefile.am
--- glame-1.0.2/src/gui/Makefile.am	Wed Apr  9 23:28:27 2003
+++ glame-1.0.2-patched/src/gui/Makefile.am	Wed Apr 23 00:17:38 2003
@@ -54,7 +54,7 @@
 	../hash/libhash.la \
 	../swapfile/libswapfile.la \
 	$(GNOME_LIBDIR) $(GLAME_LIBS) $(GNOME_LIBS) $(GNOMEUI_LIBS) $(GUILE_LIBS) \
-	$(XML_LIBS) -lm $(LIBLTDL) @LTLIBINTL@
+	$(XML_LIBS) -lm -lltdl @LTLIBINTL@
 glame_LDFLAGS = -export-dynamic -dlopen self
 
 #wavetest_SOURCES = gtkwavedraw.c gtkwavedraw.h wavetest.c
