--- glame-1.0.2/macros/acg-alsa.m4.orig	2003-05-25 23:16:36.000000000 +0200
+++ glame-1.0.2/macros/acg-alsa.m4	2004-01-16 17:45:32.691987344 +0100
@@ -55,13 +55,13 @@
 [
 $ac_have_alsa_include
 ],[
-#if SND_LIB_MAJOR != $alsa_min_major_version
+#if SND_LIB_MAJOR < $alsa_min_major_version
 #error wrong major version
 #endif
-#if SND_LIB_MINOR < $alsa_min_minor_version
+#if (SND_LIB_MAJOR == $alsa_min_major_version) && (SND_LIB_MINOR < $alsa_min_minor_version)
 #error wrong minor version
 #endif
-#if (SND_LIB_MINOR == $alsa_min_minor_version) && (SND_LIB_SUBMINOR < $alsa_min_subminor_version)
+#if (SND_LIB_MAJOR == $alsa_min_major_version) && (SND_LIB_MINOR == $alsa_min_minor_version) && (SND_LIB_SUBMINOR < $alsa_min_subminor_version)
 #error wrong subminor version
 #endif
 ],[
